odxtools/encodestate.py

```diff
@@ -83,13 +83,13 @@ def emplace_atomic_value(
 
         # Check that bytes and strings actually fit into the bit length
         if base_data_type == DataType.A_BYTEFIELD:
-            if not isinstance(internal_value, bytes):
+            if not isinstance(internal_value, (bytes, bytearray)):
```

Comment:
maybe add typing.SupportsBytes to the list as well?

Reply:
Good idea. fixed: https://github.com/mercedes-benz/odxtools/pull/298/commits/0c486ab99e064210efee6b638b627eb42bd5a49a (I'm perpetually confused about which of these type hinting helpers classes can be used for `isinstance()` and which can't.)

Reply:
The ones that have `[@runtime_checkable](https://docs.python.org/3/library/typing.html#typing.runtime_checkable).` can be used with `isinstance`